### ISSUE TITLE
I installed aka spring IDE and Spring Tool Suite, but in new ->file-》project， i can not find any spring items,who can help me ?

### ISSUE THREAD

**Issue by iver3on** [spring-projects/spring-ide#229]
I installed aka spring IDE and Spring Tool Suite, but in new ->file-》project， i can not find any spring items,who can help me ?

**Comment by martinlippert** [comment]
@iver3on What version of Eclipse did you use to install the Spring Tools into? And is there a "Spring Perspective" now available when you try to switch to a different perspective?

I guess that the overall install procedure went fine, right? No error messages?

You can also take a look at the Error Log view to see if there is anything related that could reveal more details about this.

**Comment by iver3on** [comment]
eclipse：Mars Release (4.5.0) can not install Spring Tools .
but eclipse oxygen do it.

**Comment by martinlippert** [comment]
What went wrong? Did you open the Marketplace client from within Eclipse, selected the "Spring Tools" item, clicked "install", and then? Any error message showing up?

**Comment by iver3on** [comment]
When I installed the STS, eclipse became very cat-friendly, and clicking open class file was very catchy. It's an impact experience. What's the reason？

**Comment by iver3on** [comment]
yep.In Mars Release (4.5.0)  ,I opend the Marketplace client from within Eclipse, selected the "Spring Tools" item, clicked "install".some compnents cant installed.

**Comment by martinlippert** [comment]
Can you provide a few more details? Which components can't be installed? And do you mean you open a `.class` file? Or a `.java` file? And what exactly do you mean with "catchy"? In case it gets slow, I would like to ask you to capture a few thread dumps of the running process while it is slow (using `jps` and `jstack`).

**Comment by martinlippert** [comment]
And anything in the Error Log view?
